utilities/src/apiSpecsList.json

```diff
@@ -2841,7 +2841,8 @@
       "2021-08-01-preview",
       "2021-09-01",
       "2021-12-01-preview",
-      "2022-02-01-preview"
+      "2022-02-01-preview",
```

Comment:
@ChrisSidebotham is there a reason why this file is being updated in this PR?

Reply:
@AlexanderSehr - Correct me if I am wrong but I believe the readme generator will update this file when new API Versions are available for Validation?

Reply:
It shouldn't, no. The update of the file happens automatically in a dedicated platform pipeline and you'd notice as it takes about 2-3 minutes to run its magic ;)

That being said, this update happens in many other PRs too. Given the [pipeline](https://github.com/Azure/ResourceModules/blob/main/.github/workflows/platform.apiSpecs.yml), and schedule, I guess there is a chance the workflow updates branches too?

Reply:
![image](https://user-images.githubusercontent.com/5365358/210076746-075f830c-b145-42ac-bba4-0e49a3a4cd14.png)

I'll open a dedicated issue for that. Not sure yet how to prevent it. In any case, it can be disregarded for this PR.

Reply:
Opened #2517 - @ahmadabdalla , feel free to resolve this issue